### PR TITLE
Use "beatmap" as a term for a set of difficulties (the whole osu! wiki except ranking criteria)

### PR DESCRIPTION
**Pull request by TicClick**
part of #4723

you can skip reviewing "Mapping and modding timeline" because of indentation changes, which are forced by remark

### CODE REVIEW

File: wiki/Glossary/en.md [hunk 1]
```diff
@@ -63,12 +63,6 @@ Placing an object closer than the beat spacing may trick the player into clickin
 
 To ensure consistent beat spacing, turn on the `Distance Snap` button on the right-hand side of the editor. You can choose a beat spacing value by holding `Alt` and sliding the selector at the top right until you reach the desired value. Note that the higher the value would allow a more difficult beatmap.
 
-### Beatmapset
-
-The collection of beatmaps that are packaged inside the `.osz` file. These beatmapsets are beatmaps that were created under the same mapper.
-
-Beatmapsets will always have the same music, sounds (if mapped), and video (if any) or storyboard (if any). However, do note that some beatmaps from a beatmapset may specifically contain a certain sounds, background images, or storyboards that are separate from the others.
-
```
Comment (Walavouchey) [inline wiki/Glossary/en.md:71]: Why was this removed and not repurposed as a clarification for the now-to-be-used `Beatmap` term?

File: wiki/Modding/en.md [hunk 2]
```diff
@@ -23,9 +23,9 @@ While there are countless ways to discover beatmaps to mod, the following are a
...
 
-When you come across an issue, select any relevant hit objects, then use the copy shortcut (`Ctrl` + `C`) to copy their timestamp information as text. On the beatmap's discussion, paste (`Ctrl` + `V`) any applicable timestamps and your feedback. The `Timeline` tab is used for issues regarding specific parts of a beatmap, the `General (This difficulty)` tab is used for feedback on larger sections of a beatmap, and the `General (All difficulties)` tab is used for issues applicable to all difficulties on a beatmapset.
+When you come across an issue, select any relevant hit objects, then use the copy shortcut (`Ctrl` + `C`) to copy their timestamp information as text. On the beatmap's discussion, paste (`Ctrl` + `V`) any applicable timestamps and your feedback. The `Timeline` tab is used for issues regarding specific parts of a beatmap, the `General (This difficulty)` tab is used for feedback on larger sections of a beatmap, and the `General (All difficulties)` tab is used for issues applicable to all difficulties on a beatmap.
```
Comment (Walavouchey) [inline wiki/Modding/en.md:28]: small nit for consistency

```suggestion
When you come across an issue, select any relevant hit objects, then use the copy shortcut (`Ctrl` + `C`) to copy their timestamp information as text. On the beatmap's discussion, paste (`Ctrl` + `V`) any applicable timestamps and your feedback. The `Timeline` tab is used for issues regarding specific parts of a beatmap, the `General (This difficulty)` tab is used for feedback on larger sections of a beatmap, and the `General (All difficulties)` tab is used for issues applicable to all difficulties of a beatmap.
```

File: wiki/People/The_Team/Beatmap_Nominators/General_Information/en.md [hunk 3]
```diff
@@ -18,15 +18,15 @@ Nominations and resetting nominations are the evidence of a BN's efforts when ch
...
+**This button allows you to nominate beatmaps.** To nominate a beatmap, the mapset must have at least 5 Hype and no currently unresolved Problem or Suggestion stamps. Make sure you're satisfied with all difficulties, even other game modes, before placing this. If the beatmap was previously vetoed, you must ensure the veto is resolved before moving a mapset forward. When a beatmap has 2 nominations, it will be Qualified and enter the ranking queue.
 
 ### Disqualify or reset nomination 
 
 ![disqualify and reset buttons](img/reset_dq.png)
 
 **This allows you to reset a beatmap's progress in the ranking proccess.** If a beatmap is currently nominated or qualified, you as a nominator are given the ability to reset the beatmaps' nomination(s) by posting an issue using a Problem stamp. If a beatmap is qualified, it can only be disqualified by a full BN. These options are usually used when there is a major or unrankable issue in the beatmap which should be addressed before considering it for ranking.
 
-Steps on Disqualifying a Beatmapset:
+Steps on disqualifying a beatmap:
```
Comment (Walavouchey) [inline wiki/People/The_Team/Beatmap_Nominators/General_Information/en.md:29]:
```suggestion
Steps for disqualifying a beatmap:
```

File: wiki/People/The_Team/Beatmap_Nominators/en.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ Full Beatmap Nominators with below minimum nomination activity will not be place
 
 ### Probationary Beatmap Nominators
 
-Probation is used to monitor new or concerning Beatmap Nominators more closely. The main role for probationary Beatmap Nominators is to establish a foundation as a Beatmap Nominator through exclusively promoting beatmaps in the ranking process. At least one full nominator must be involved in the ranking process for each game mode on a beatmapset, so probationary members cannot give the final nomination to a beatmap which has only received nominations from other probationary members for their game mode.
+Probation is used to monitor new or concerning Beatmap Nominators more closely. The main role for probationary Beatmap Nominators is to establish a foundation as a Beatmap Nominator through exclusively promoting beatmaps in the ranking process. At least one full nominator must be involved in the ranking process for each game mode on a beatmap, so probationary members cannot give the final nomination to a beatmap which has only received nominations from other probationary members for their game mode.
```
Comment (Walavouchey) [inline wiki/People/The_Team/Beatmap_Nominators/en.md:31]:
```suggestion
Probation is used to monitor new or concerning Beatmap Nominators more closely. The main role for probationary Beatmap Nominators is to establish a foundation as a Beatmap Nominator through exclusively promoting beatmaps in the ranking process. At least one full nominator must be involved in the ranking process for each game mode of a beatmap, so probationary members cannot give the final nomination to a beatmap which has only received nominations from other probationary members for their game mode.
```

**Review by Walavouchey — COMMENTED** [review]
Checked the whole English side - seems fine. Just noticed a few miniscule things.

Will check Finnish and Swedish shortly.

File: wiki/Installation/fi.md [hunk 5]
```diff
@@ -25,7 +25,7 @@ Tämä sivu kertoo miten saat osu!'n toimimaan Windows-tietokoneellesi. [osu!aka
    - Jos sinulla on tili, kirjaudu sisään.
    - Jos tarvitset rekisteröityä, mene [rekisteröitymissivulle](/wiki/Registration).
 
-## Beatmapsettien lataaminen
+## Beatmaptien lataaminen
```
Comment (Walavouchey) [inline wiki/Installation/fi.md:28]:
```suggestion
## Beatmappien lataaminen
```

File: wiki/Glossary/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/Glossary/en.md:71]: there are two reasons for this:

- we have https://github.com/ppy/osu-wiki/tree/master/wiki/Beatmap/ for that
- the glossary in its current form (a large page) will be removed in favor of small and specific articles: #4804

**Review by Walavouchey — CHANGES_REQUESTED** [review]
Nvm, there wasn't much to check

File: wiki/Beatmap_Discussion/fr.md [hunk 6]
```diff
@@ -134,7 +134,7 @@ Si vous postez dans l'onglet `Chronologie`, vous devez inclure un horodatage dan
...
+Le créateur du beatmap et l'auteur d'un modpost, ont la possibilité de fermer les problèmes avec le bouton `Marquer comme résolu`. Cela indique aux autres utilisateurs que le problème a été traité et le supprime de la liste des questions dans [l'option de tri](#options-de-tri) `En Attente`. Les modposts peuvent être réouverts par n'importe quel utilisateur en ajoutant des réponses supplémentaires à un post et en cliquant sur `Répondre et réouvrir`. Ceci est utilisé lorsque les moddeurs tombent sur des problèmes fermés qui n'ont pas été entièrement résolus, ou lorsqu'un sujet nécessite une discussion plus approfondie.
```
Comment (DoryanR) [inline wiki/Beatmap_Discussion/fr.md:137]:
```suggestion
Le créateur de la beatmap et l'auteur d'un modpost, ont la possibilité de fermer les problèmes avec le bouton `Marquer comme résolu`. Cela indique aux autres utilisateurs que le problème a été traité et le supprime de la liste des questions dans [l'option de tri](#options-de-tri) `En Attente`. Les modposts peuvent être réouverts par n'importe quel utilisateur en ajoutant des réponses supplémentaires à un post et en cliquant sur `Répondre et réouvrir`. Ceci est utilisé lorsque les moddeurs tombent sur des problèmes fermés qui n'ont pas été entièrement résolus, ou lorsqu'un sujet nécessite une discussion plus approfondie.
```

File: wiki/Beatmap_Editor/Song_Setup/fr.md [hunk 7]
```diff
@@ -16,7 +16,7 @@ Song Setup est la quatrième section de [Beatmap Editor](/wiki/Beatmap_Editor).
 
 ![A completed Song Setup form](img/SS_General.jpg "Un exemple de métadonnées complètes (onglet Général) dans la configuration de la chanson")
 
-Ces champs doivent être remplis avant de créer une deuxième difficulté. Ceci est principalement pour s'assurer que les métadonnées seront cohérentes à travers toutes les difficultés du beatmapset.
+Ces champs doivent être remplis avant de créer une deuxième difficulté. Ceci est principalement pour s'assurer que les métadonnées seront cohérentes à travers toutes les difficultés du beatmap.
```
Comment (DoryanR) [inline wiki/Beatmap_Editor/Song_Setup/fr.md:19]:
```suggestion
Ces champs doivent être remplis avant de créer une deuxième difficulté. Ceci est principalement pour s'assurer que les métadonnées seront cohérentes à travers toutes les difficultés de la beatmap.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 8]
```diff
@@ -15,11 +15,11 @@ Le [modding](/wiki/Modding) est le processus qui consiste à recevoir des critiq
...
 
-Avant d'être nominés pour le classement, les beatmapsets doivent recevoir [hype](/wiki/Glossary#hype) des messages d'au moins cinq utilisateurs différents.
+Avant d'être nominés pour le classement, les beatmaps doivent recevoir [hype](/wiki/Glossary#hype) des messages d'au moins cinq utilisateurs différents.
 
 ## Nominations
 
-Les membres des [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BNs*) sont des modders expérimentés qui sont capables de nommer des beatmapsets pour le classement. Leur rôle est de s'assurer qu'un beatmapset répond aux critères du [Ranking Criteria](/wiki/Ranking_Criteria) ainsi que leurs propres normes de qualité personnelles.
+Les membres des [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BNs*) sont des modders expérimentés qui sont capables de nommer des beatmaps pour le classement. Leur rôle est de s'assurer qu'un beatmap répond aux critères du [Ranking Criteria](/wiki/Ranking_Criteria) ainsi que leurs propres normes de qualité personnelles.
```
Comment (DoryanR) [inline wiki/Beatmap_ranking_procedure/fr.md:22]:
```suggestion
Les membres des [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BNs*) sont des modders expérimentés qui sont capables de nommer des beatmaps pour le classement. Leur rôle est de s'assurer qu'une beatmap répond aux critères du [Ranking Criteria](/wiki/Ranking_Criteria) ainsi que leurs propres normes de qualité personnelles.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 9]
```diff
@@ -33,20 +33,20 @@ Cette catégorie de beatmaps existe pour fournir une plus grande exposition à l
...
 
-Lorsque une beatmapset est disqualifié, sa position dans la file d'attente de classement est sauvegardée. S'il est re-qualifié, il sera placé à sa position précédente dans la file d'attente de classement. Par exemple, si un beatmapset est qualifié pendant 2 jours, puis disqualifié et re-qualifié, il restera dans la catégorie Qualifié pendant environ 5 jours de plus. Si la position sauvegardée dans la file d'attente est à moins de 24 heures du classement, elle sera remise à 24 heures lors de l'entrée dans la catégorie Qualifié.
+Lorsque une beatmap est disqualifié, sa position dans la file d'attente de classement est sauvegardée. S'il est re-qualifié, il sera placé à sa position précédente dans la file d'attente de classement. Par exemple, si un beatmap est qualifié pendant 2 jours, puis disqualifié et re-qualifié, il restera dans la catégorie Qualifié pendant environ 5 jours de plus. Si la position sauvegardée dans la file d'attente est à moins de 24 heures du classement, elle sera remise à 24 heures lors de l'entrée dans la catégorie Qualifié.
```
Comment (DoryanR) [inline wiki/Beatmap_ranking_procedure/fr.md:38]:
```suggestion
Lorsque une beatmap est disqualifié, sa position dans la file d'attente de classement est sauvegardée. S'il est re-qualifié, il sera placé à sa position précédente dans la file d'attente de classement. Par exemple, si une beatmap est qualifié pendant 2 jours, puis disqualifié et re-qualifié, il restera dans la catégorie Qualifié pendant environ 5 jours de plus. Si la position sauvegardée dans la file d'attente est à moins de 24 heures du classement, elle sera remise à 24 heures lors de l'entrée dans la catégorie Qualifié.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 10]
```diff
@@ -33,20 +33,20 @@ Cette catégorie de beatmaps existe pour fournir une plus grande exposition à l
...
 
-Les nominations peuvent être réinitialisées quand le créateur d'un beatmapset met à jour l'ensemble ou quand un membre du BN ou du NAT trouve un problème dans un beatmapset nommé. Elles peuvent aussi être réinitialisées par les membres de la [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) quand la raison implique un contenu inapproprié. Si un beatmapset est qualifié lorsque ses nominations sont réinitialisées, l'ensemble sera disqualifié, le retirant de la file de classement.
+Les nominations peuvent être réinitialisées quand le créateur d'un beatmap met à jour l'ensemble ou quand un membre du BN ou du NAT trouve un problème dans un beatmap nommé. Elles peuvent aussi être réinitialisées par les membres de la [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) quand la raison implique un contenu inapproprié. Si un beatmap est qualifié lorsque ses nominations sont réinitialisées, l'ensemble sera disqualifié, le retirant de la file de classement.
```
Comment (DoryanR) [inline wiki/Beatmap_ranking_procedure/fr.md:44]:
```suggestion
Les nominations peuvent être réinitialisées quand le créateur d'une beatmap met à jour l'ensemble ou quand un membre du BN ou du NAT trouve un problème dans une beatmap nommé. Elles peuvent aussi être réinitialisées par les membres de la [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) quand la raison implique un contenu inapproprié. Si une beatmap est qualifié lorsque ses nominations sont réinitialisées, l'ensemble sera disqualifié, le retirant de la file de classement.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 11]
```diff
@@ -33,20 +33,20 @@ Cette catégorie de beatmaps existe pour fournir une plus grande exposition à l
...
 
 Membres du BN et du NAT à l'occasion [veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) les ensembles de battants qu'ils estiment avoir des problèmes de qualité qui ne sont pas liés aux critères de classement.
 
 ## Classé
 
-Les Beatmapsets qui sont passés par l'étape de qualification ont terminé la procédure de classement. Ils ont des tableaux de classement et récompensent les joueurs avec des points de performance.
+Les Beatmaps qui sont passés par l'étape de qualification ont terminé la procédure de classement. Ils ont des tableaux de classement et récompensent les joueurs avec des points de performance.
```
Comment (DoryanR) [inline wiki/Beatmap_ranking_procedure/fr.md:50]:
```suggestion
Les beatmaps qui sont passés par l'étape de qualification ont terminé la procédure de classement. Ils ont des tableaux de classement et récompensent les joueurs avec des points de performance.
```

File: wiki/Game_mode/osu!catch/fr.md [hunk 12]
```diff
@@ -213,5 +213,5 @@ Le système utilisé pour calculer la perte ou le gain de vie est compliqué et
...
 
 - Jusqu’au 10 avril 2012 (10/04/2012), les maps osu!catch ne pouvaient pas être ranked ou être inclus dans les maps ranked. Néanmois, cela a changé, et les mappeurs peuvent maintenant créer des difficultées spécialisée osu!catch, à condition qu'ils suivent les directives. Ces directives sont en cours d’écriture, mais plus de maps osu!catch sont placées en ranked et un feedback est donné par les joueurs, les directives évolueront pour être aussi complet et exhaustif que les directives osu! standard.
-- Le premier beatmapset ranked qui contient au moins une beatmap osu!catch spécifique est [Yousei Teikoku - Dare so Ka no Gekka (TV Size) (NatsumeRin), avec la beatmap osu!catch invitée mappée par Uran](https://osu.ppy.sh/beatmapsets/13676)
-- Le premier beatmapset ranked composé uniquement de maps osu!catch est [Rita - Hajimari no Toki (Deif)](https://osu.ppy.sh/beatmapsets/91485)
+- Le premier beatmap ranked qui contient au moins une difficultée osu!catch spécifique est [Yousei Teikoku - Dare so Ka no Gekka (TV Size) (NatsumeRin), avec la beatmap osu!catch invitée mappée par Uran](https://osu.ppy.sh/beatmapsets/13676)
```
Comment (DoryanR) [inline wiki/Game_mode/osu!catch/fr.md:216]:
```suggestion
- La première beatmap ranked qui contient au moins une difficultée osu!catch spécifique est [Yousei Teikoku - Dare so Ka no Gekka (TV Size) (NatsumeRin), avec la beatmap osu!catch invitée mappée par Uran](https://osu.ppy.sh/beatmapsets/13676)
```

File: wiki/Game_mode/osu!catch/fr.md [hunk 13]
```diff
@@ -213,5 +213,5 @@ Le système utilisé pour calculer la perte ou le gain de vie est compliqué et
...
+- Le premier beatmap ranked composé uniquement de maps osu!catch est [Rita - Hajimari no Toki (Deif)](https://osu.ppy.sh/beatmapsets/91485)
```
Comment (DoryanR) [inline wiki/Game_mode/osu!catch/fr.md:217]:
```suggestion
- La première beatmap ranked composé uniquement de maps osu!catch est [Rita - Hajimari no Toki (Deif)](https://osu.ppy.sh/beatmapsets/91485)
```

File: wiki/Game_mode/osu!taiko/fr.md [hunk 14]
```diff
@@ -255,5 +255,5 @@ Notez également que les drumrolls en 1/6 sont donnés si le taux de slider tick
 ![logo osu!taiko en modes spécial](img/Taiko_logo.jpg "Logo Taiko dans un mode spécial")
 
 - Le`Use Taiko skin for Taiko mode` dans la barre sur le côté en dessous de la section skin va utiliser les éléments du skin dans le dossier `taiko` quand vous jouez à osu!taiko, sans prendre en compte les éléments taiko du skin actuel. Le dossier utilisé pour contenir le skin *[Taiko by LuiginHann](https://osu.ppy.sh/community/forums/topics/41319)*, qui peut être téléchargé par le maintenant déprécié `osume.exe` (un exécutable permettant de mettre osu! à jour quand il en avait besoin) sous la catégorie `Skin`.
-- Le premier beatmapset ranked qui contenait au moins une beatmap spécifique à osu!taiko est [Taiko no Tatsujin - Saitama2000 (Kharl)](https://osu.ppy.sh/beatmapsets/210)
-- Le premier beatmapset ranked avec des beatmap uniquement osu!taiko est [Mutsuhiko Izumi - Red Goose (lepidopodus)](https://osu.ppy.sh/beatmapsets/55920)
+- Le premier beatmap ranked qui contenait au moins une difficultée spécifique à osu!taiko est [Taiko no Tatsujin - Saitama2000 (Kharl)](https://osu.ppy.sh/beatmapsets/210)
```
Comment (DoryanR) [inline wiki/Game_mode/osu!taiko/fr.md:258]:
```suggestion
- La première beatmap ranked qui contenait au moins une difficultée spécifique à osu!taiko est [Taiko no Tatsujin - Saitama2000 (Kharl)](https://osu.ppy.sh/beatmapsets/210)
```

File: wiki/Guides/BSS_Issues/fr.md [hunk 15]
```diff
@@ -24,9 +24,9 @@ Si vous avez toujours des problèmes, suivez ces étapes :
 
 1. Exportez la beatmap en utilisant l’éditeur de beatmap.
    - `File` puis `Export Package`.
-2. Sortez de l’éditeur de beatmap et supprimez le beatmapset depuis osu!.
+2. Sortez de l’éditeur de beatmap et supprimez le beatmap depuis osu!.
 3. Sortez d’osu!
-4. Allez dans le beatmapset exporté et changez l’extension de `.osz` à `.zip` (vous devez avoir les extensions de fichier visible dans l’explorateur de fichier pour faire ceci).
+4. Allez dans le beatmap exporté et changez l’extension de `.osz` à `.zip` (vous devez avoir les extensions de fichier visible dans l’explorateur de fichier pour faire ceci).
```
Comment (DoryanR) [inline wiki/Guides/BSS_Issues/fr.md:29]:
```suggestion
4. Allez dans la beatmap exporté et changez l’extension de `.osz` à `.zip` (vous devez avoir les extensions de fichier visible dans l’explorateur de fichier pour faire ceci).
```

File: wiki/Installation/fr.md [hunk 16]
```diff
@@ -49,7 +49,7 @@ Il existe deux manières d'ajouter des beatmaps, via le site Web ou osu!direct (
 2. Trouvez une chanson que vous aimeriez.
 3. Double-cliquez sur la chanson dans la liste ou cliquez sur `Télécharger` à droite pour la télécharger.
    - Si vous ne souhaitez pas télécharger la vidéo, cliquez sur `DL NoVideo` en jeu.
-4. Quand osu! a fini le téléchargement du beatmapset, il sera automatiquement chargé si vous êtes en mode solo, à l'écran [Song Selection](/wiki/Interface#song-selection-screen).
+4. Quand osu! a fini le téléchargement du beatmap, il sera automatiquement chargé si vous êtes en mode solo, à l'écran [Song Selection](/wiki/Interface#song-selection-screen).
```
Comment (DoryanR) [inline wiki/Installation/fr.md:52]:
```suggestion
4. Quand osu! a fini le téléchargement de la beatmap, il sera automatiquement chargé si vous êtes en mode solo, à l'écran [Song Selection](/wiki/Interface#song-selection-screen).
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
Fix for French files :

File: wiki/Shortcut_key_reference/en.md [hunk 17]
```diff
@@ -71,7 +71,7 @@ These shortcuts work anywhere:
 | `Shift` + `F2` | Go back to the previously random-selected beatmap. |
 | `F3` | Open the beatmap options. |
 | `F5` | Reprocess beatmaps (this may prompt you to continue). |
-| `Shift` + `Delete` | Prompt the delete beatmap dialog. This will ask you if you want to delete the beatmapset (all difficulties), the beatmap (the selected difficulty), or all visible beatmaps. |
+| `Shift` + `Delete` | Prompt the delete beatmap dialog. This will ask you if you want to delete the beatmap (all difficulties), a single difficulty, or all visible difficulties. |
```
Comment (TPGPL) [inline wiki/Shortcut_key_reference/en.md:74]:
```suggestion
| `Shift` + `Delete` | Prompt the delete beatmap dialog. This will ask you if you want to delete the beatmap (all difficulties), a single difficulty, or all visible beatmaps. |
```

**Comment by TicClick** [comment]
@TPGPL I've accidentally deleted your suggestion for the osu!catch article while trying to slightly adjust it, could you repost it again?

File: wiki/Game_mode/osu!catch/fr.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (TicClick) [reply wiki/Game_mode/osu!catch/fr.md]: @DoryanR could you handle this part, please?

File: wiki/Game_mode/osu!catch/fr.md [hunk 18]
```diff
@@ -213,5 +213,5 @@ Le système utilisé pour calculer la perte ou le gain de vie est compliqué et
...
 
 - Jusqu’au 10 avril 2012 (10/04/2012), les maps osu!catch ne pouvaient pas être ranked ou être inclus dans les maps ranked. Néanmois, cela a changé, et les mappeurs peuvent maintenant créer des difficultées spécialisée osu!catch, à condition qu'ils suivent les directives. Ces directives sont en cours d’écriture, mais plus de maps osu!catch sont placées en ranked et un feedback est donné par les joueurs, les directives évolueront pour être aussi complet et exhaustif que les directives osu! standard.
-- Le premier beatmapset ranked qui contient au moins une beatmap osu!catch spécifique est [Yousei Teikoku - Dare so Ka no Gekka (TV Size) (NatsumeRin), avec la beatmap osu!catch invitée mappée par Uran](https://osu.ppy.sh/beatmapsets/13676)
-- Le premier beatmapset ranked composé uniquement de maps osu!catch est [Rita - Hajimari no Toki (Deif)](https://osu.ppy.sh/beatmapsets/91485)
+- La première beatmap ranked qui contient au moins une difficultée osu!catch spécifique est [Yousei Teikoku - Dare so Ka no Gekka (TV Size) (NatsumeRin), avec la beatmap osu!catch invitée mappée par Uran](https://osu.ppy.sh/beatmapsets/13676)
```
Comment (DoryanR) [inline wiki/Game_mode/osu!catch/fr.md:216]:
```suggestion
- La première beatmap classé qui contient au moins une difficulté osu!catch est [Yousei Teikoku - Dare so Ka no Gekka (TV Size) (NatsumeRin), avec une guest difficulty osu!catch mappé par Uran](https://osu.ppy.sh/beatmapsets/13676).
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]
we can go with this i think

**Review by Ephemeralis — APPROVED** [review]
